BUILDING.md

```diff
@@ -33,7 +33,7 @@ file a new issue.
     * [Speeding up frequent rebuilds when developing](#speeding-up-frequent-rebuilds-when-developing)
     * [Troubleshooting Unix and macOS builds](#troubleshooting-unix-and-macos-builds)
   * [Windows](#windows)
-    * [Prerequisites](#prerequisites)
+    * [Windows Prerequisites](#windows-prerequisites)
```

Comment:
I think that this link is broken, at least in the GitHub UI. The correct fragment is `#prerequisites-1`, [see](https://github.com/nodejs/node/blob/51931d11be8eeec1af52550916fb2ed4d9f0ae76/BUILDING.md#prerequisites-1)

I suggest to change the heading and keep the reference as `#windows-prerequisites`.

Reply:
Oh I missed updating the heading it seems. I'll fix that now.